cookbooks/aws-parallelcluster-install/recipes/base.rb

```diff
@@ -146,27 +146,7 @@
 # Check whether install a custom aws-parallelcluster-node package or the standard one
 if !node['cluster']['custom_node_package'].nil? && !node['cluster']['custom_node_package'].empty?
   # Install custom aws-parallelcluster-node package
-  bash "install aws-parallelcluster-node" do
-    cwd Chef::Config[:file_cache_path]
-    code <<-NODE
-      set -e
-      [[ ":$PATH:" != *":/usr/local/bin:"* ]] && PATH="/usr/local/bin:${PATH}"
-      echo "PATH is $PATH"
-      source #{node['cluster']['node_virtualenv_path']}/bin/activate
-      pip uninstall --yes aws-parallelcluster-node
-      if [[ "#{node['cluster']['custom_node_package']}" =~ ^s3:// ]]; then
-        custom_package_url=$(#{node['cluster']['cookbook_virtualenv_path']}/bin/aws s3 presign #{node['cluster']['custom_node_package']} --region #{node['cluster']['region']})
-      else
-        custom_package_url=#{node['cluster']['custom_node_package']}
-      fi
-      curl --retry 3 -L -o aws-parallelcluster-node.tgz ${custom_package_url}
-      mkdir aws-parallelcluster-custom-node
-      tar -xzf aws-parallelcluster-node.tgz --directory aws-parallelcluster-custom-node
-      cd aws-parallelcluster-custom-node/*aws-parallelcluster-node-*
-      pip install .
-      deactivate
-    NODE
-  end
+  include_recipe 'aws-parallelcluster-install::parallelcluster_custom_node'
```

Comment:
I'd remove `_custom_`, we are installing `parallelcluster-node`

Reply:
nevermind, I see that only the portion to install custom node was moved.
Can we move everything in the new cookbook so we have only one place where the node is installed?